graylog2-web-interface/package.json

```diff
@@ -73,48 +78,46 @@
     "lodash": "^4.17.4",
     "markdown": "^0.5.0",
     "md5": "^2.2.1",
+    "moment": "^2.24.0",
     "moment-duration-format": "2.2.2",
     "numeral": "^2.0.6",
     "opensans-npm-webfont": "^1.0.0",
+    "plotly.js": "^1.48.1",
     "qs": "^6.3.0",
     "react-ace-builds": "^7.2.4",
     "react-color": "^2.14.0",
     "react-day-picker": "^5.0.0",
     "react-dnd": "^7.0.2",
     "react-dnd-html5-backend": "^2.0.0",
     "react-grid-layout": "^0.16.6",
+    "react-immutable-proptypes": "^2.1.0",
     "react-isolated-scroll": "^0.1.1",
     "react-leaflet": "^2.3.0",
     "react-overlays": "^0.7.0",
+    "react-plotly.js": "^2.3.0",
+    "react-portal": "^4.2.0",
     "react-resizable": "^1.8.0",
     "react-select": "2.4.3",
+    "react-sortable-hoc": "^1.7.1",
+    "react-window": "^1.8.2",
     "rickshaw": "^1.5.1",
     "sockjs-client": "1.1.x",
     "superagent": "^3.4.3",
     "superagent-bluebird-promise": "^4.1.0",
     "toastr": "^2.1.2",
+    "twix": "^1.1.4",
     "typeahead.js": "^0.11.1",
     "ua-parser-js": "^0.7.12",
     "uglifyjs-webpack-plugin": "2.0.1",
     "urijs": "^1.19.1",
-    "bson-objectid": "^1.2.5",
-    "chroma-js": "^2.0.3",
-    "moment": "^2.24.0",
-    "plotly.js": "^1.48.1",
-    "react-immutable-proptypes": "^2.1.0",
-    "react-plotly.js": "^2.3.0",
-    "react-portal": "^4.2.0",
-    "react-sortable-hoc": "^1.7.1",
-    "react-window": "^1.8.2",
-    "twix": "^1.1.4",
     "uuid": "^3.2.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.4.5",
-    "@babel/plugin-proposal-class-properties": "^7.4.4",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.4.5",
+    "@babel/core": "7.5.0",
+    "@babel/plugin-proposal-class-properties": "7.5.0",
+    "@babel/plugin-syntax-dynamic-import": "7.2.0",
+    "@babel/polyfill": "7.4.4",
```

Comment:
https://babeljs.io/docs/en/babel-polyfill

> As of Babel 7.4.0, this package has been deprecated in favor of directly including core-js/stable (to polyfill ECMAScript features) and regenerator-runtime/runtime (needed to use transpiled generator functions)

> Because this is a polyfill (which will run before your source code), we need it to be a dependency, not a devDependency